packages/runtime/container-runtime/src/orderedClientElection.ts

```diff
@@ -364,23 +460,39 @@ export class OrderedClientElection
     }
 
     public incrementElectedClient(sequenceNumber: number): void {
-        const nextClient = this.findFirstEligibleClient(this._electedClient?.youngerClient);
-        this.tryElectingClient(nextClient, sequenceNumber);
+        const nextClient = this.findFirstEligibleParent(this._electedParent?.youngerClient) ??
```

Comment:
`this.findFirstEligibleParent(this._electedParent?.youngerClient) ?? this.findFirstEligibleParent(this.orderedClientCollection.oldestClient)` is done a few times. What does the two calls do? Also, maybe this needs to be a function called something like `findNextEligibleParent`?

Reply:
The first call tries to find the oldest eligible client that is younger than _electedParent. There may not be one, so then it will start over with the oldest client in the quorum.